File: NN_DASH/nn_dash/app/src/main/java/com/example/nn_dashboard/Product.java
package com.example.nn_dashboard;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "tbl_products")
public class Product {
    @PrimaryKey(autoGenerate = true)
    public int prod_id;
    public int cat_id;
    public String prod_name;
    public String prod_image;
    public String prod_disc;
    public int price;
    public int prod_status;
}

File: DASH-ROOM/test1_dash_room/app/src/main/java/com/example/test1_dash_room/Product.kt
package com.example.test1_dash_room

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "tbl_products")
data class Product(
    @PrimaryKey(autoGenerate = true) val prod_id: Int = 0,
    val cat_id: Int,
    val prod_name: String,
    val prod_image: String,
    val prod_desc: String,
    val price: Int,
    val prod_status: Double
)